src/test/scala/com/cognite/sdk/scala/common/StringDataPointsResourceBehaviors.scala

```diff
@@ -7,15 +7,15 @@ import org.scalatest.{FlatSpec, Matchers}
 
 trait StringDataPointsResourceBehaviors[I] extends Matchers { this: FlatSpec =>
   private val startTime = System.currentTimeMillis()
-  private val endTime = startTime + 20 * 1000
-  private val testStringDataPoints = (startTime to endTime by 1000)
-    .map(t => StringDataPoint(Instant.ofEpochMilli(t), math.random.toString))
+  private val endTime = startTime + 20*1000
+  private val testStringDataPoints = (startTime to endTime by 1000).map(t =>
+    StringDataPoint(Instant.ofEpochMilli(t), math.random.toString))
 
-  def withStringTimeSeriesId(testCode: I => Any): Unit
+  def withStringTimeSeriesId(testCode: (I, I) => Any): Unit
 
   def stringDataPointsResource(dataPoints: DataPointsResource[Id, I]): Unit =
     it should "be possible to insert and delete string data points" in withStringTimeSeriesId {
-      stringTimeSeriesId =>
+      (stringTimeSeriesId, missingStringTimeSeriesId) =>
```

Comment:
We don't need to specify this, just hard code it to be some large random number - the likelihood of us hitting that large random number is close enough to zero that I'm willing to take the chance in exchange for simpler code.